Casks/noports.rb

```diff
@@ -44,17 +47,14 @@
 
   on_macos do
     service "sshnp/launchd/com.atsign.sshnpd.plist"
+    artifact "sshnp/config/sshnpd.yaml",
+      target: "~/Library/Application Support/Noports/sshnpd.yaml"
   end
 
   on_linux do
     service "sshnp/systemd/sshnpd.service"
-    service "sshnp/systemd/srvd.service"
```

Comment:
Removed srvd. If you are planning on running a relay, you shouldn't be using brew as your package manager.